doc/BASECAMP.md

```diff
@@ -140,7 +140,7 @@ There are two special recipe groups, `"all_faction_base_types"` and `"all_factio
 
 Each entry in the `"recipes"` array must be a dictionary with the `"id"`, `"description"`, and `"om_terrains"` fields.  `"id"` is the recipe `"id"` of the recipe that starts that basecamp or basecamp expansion upgrade path.  `"description"` is a short name of the basecamp or basecamp expansion.  `"om_terrains"` is a list of overmap terrain ids which can be used as the basis for the basecamp or basecamp expansion.
 
-All recipes that start an upgrade path or expansion should have a blueprint requirement that can never be met, such as "not_an_upgrade", to prevent them from showing up as available upgrades.
+All recipes that start an upgrade path or expansion should have a blueprint requirement that can never be met, such as "not_an_upgrade", to prevent them from showing up as available upgrades. Additionally, if you want to add expansion, you must create OMT with same `id` as expansions's `id`.
```

Comment:
```suggestion
All recipes that start an upgrade path or expansion should have a blueprint requirement that can never be met, such as "not_an_upgrade", to prevent them from showing up as available upgrades.  Additionally, if you want to add an expansion, you must create an OMT with the same `id` as the expansion's `id`.
```

Reply:
There are quite a few arcane restrictions involved in base camp expansions. Apart from the OMT, you also need a graphic display definition in another file. The name of the expansion also has to match a specific pattern where the suffix is fixed. I think there are additional gotchas of a similar type.

Reply:
What do you mean by graphic display definition?

Reply:
You have to specify what "character" to use to display the tile on the OMT ("character" being actual character plus color and background). The reason I'm vague on this is that I did my work on it too many months ago for it to be fresh in memory.

Reply:
I see, I tought you mean something else.